### PR TITLE
Switch to webpack

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: scripts/travis_script.sh [hunk 1]
```diff
@@ -6,4 +6,6 @@ npm run clean
 npm run build
 npm test
 npm run test:coverage
+npm run clean:example
```
Comment (sccolbert) [inline scripts/travis_script.sh]: you don't need the `clean` line here.

Comment (blink1073) [inline scripts/travis_script.sh]: I put it in there to make sure we weren't blowing away the webpack.conf.js file, which I had done originally :)

Comment (sccolbert) [inline scripts/travis_script.sh]: I'm not seeing how those two things are related. The Travis install starts empty, so the `clean` is effectively a no-op.

Comment (blink1073) [inline scripts/travis_script.sh]: It isn't empty, the contents of the repo are there (which includes webpack.conf.js).  When it was being blown away, `npm run build:example` was failing.

Comment (blink1073) [inline scripts/travis_script.sh]: fixed

**Pull request merged**